tasks/queue-granules/tests/index.js

```diff
@@ -1065,3 +1065,17 @@ test.serial('queueGranules throws an error when no dataType, version, or collect
 
   await t.throwsAsync(queueGranules(event));
 });
+
+test.serial('queueGranules does not throw an error when updatedAt is provided and is not provided', async (t) => {
```

Comment:
If we're including the happy path case, we definitely should include the negative path we're actually trying to prevent, e.g. test for using a string failing schema validation. 

There's a pattern question here as to if we want to explicitly test the schema definitions explicitly in units, but as a fix to the immediate problem this is fine.